src/layout/InstantiationButton/InstantiationButtonComponent.tsx

```diff
@@ -16,18 +16,16 @@ const rowStyle = {
 
 export function InstantiationButtonComponent({ text, ...props }: IInstantiationButtonComponentProps) {
   return (
-    <div className='container pl-0'>
+    <div
+      className='a-btn-group'
+      style={btnGroupStyle}
+    >
       <div
-        className='a-btn-group'
-        style={btnGroupStyle}
+        className='row'
+        style={rowStyle}
```

Comment:
What do you think about creating a CSS module for this component instead of using global CSS? Then we do not need the inline styling. I needed to think twice about why it should be necessary to set `marginLeft: 0`, because that should be the default on a `<div/>` tag. Then I recognized that the global row class have other values than preferred for this component. What do you think? 😄 
```suggestion
        className={classes.row}
```

I think it would be nice if we do not have any global CSS for such things, but instead have it in CSS modules. Then we achieve separation of concerns. I think it would be easier to maintain the CSS within the project if it's not global. If we need some spacing, and theming that is the same throughout our components, we could use CSS variables for that.